src/signal.rs

```diff
@@ -30,7 +30,7 @@ pub enum Error {
     /// Couldn't create a sigset.
     CreateSigset(errno::Error),
     /// The wrapped signal has already been blocked.
-    SignalAlreadyBlocked(c_int),
+    SignalAlreadyBlocked(i32),
```

Comment:
Sorry for not expressing myself clearly in the previous comment about `c_int` vs `i32` and `c_int` possibly being different on different platforms; I meant sticking with `c_int` (especially given that we apparently have stuff like `c_void` scattered around as well). At the same time, leaving the changes as they are right now in this commit does not do any harm. We can revisit the decision later if reverting them is too bothersome.

Reply:
Confusing now ;) I would like to remove the last commit in this PR(This means there will have both c_int and i32 cases), and change i32 to c_int, as my previous comment. Is this the same as your suggestion?